plugins/logs/service_events

```diff
@@ -330,10 +368,17 @@ function fetch() {
         logmatch="${LOGFILEMAP[$n]}_regex"
         matches="$(tail -n +"$prvlines" "$logfile" | grep -Ec "${!logmatch}" || true)"
 
-        # Aggregate and add to the correct service counter
-        svc_counter="${svcnm}_total"
-        !((matches+=${!svc_counter}))
-        typeset "$svc_counter=$matches"
+        # If there were matches, aggregate them and add this log to the extinfo for the service
+        if [ "$matches" -gt 0 ]; then
+            # Aggregate and add to the correct service counter
+            svc_counter="${svcnm}_total"
```

Comment:
Maybe clarify withe name of the variable, that it is a *name* instead of a *value*?

Reply:
+1